### DISCUSSION TITLE
!! Arbeiten am Bogen werden eingestellt !!

### DISCUSSION THREAD

**Discussion by Gorthian** [Gorthian/HeXXen1733_Custom_Character_Sheet#49] [General]
Ulisses hat Anfang 2022 still und heimlich einen offiziellen Bogen bei Roll20 veröffentlicht.
Da dieser hervorragend gelungen ist, werde ich die Arbeiten an meinem eigenen Bogen einstellen.

**Comment by runeb68** [comment]
Schade, trotzdem vielen Dank.

Wir bleiben vermutlich erst einmal bei Deinem Bogen. 

Was mir fehlt ist die Möglichkeit die Steigerungen festzuhalten ähnlich wie auf dem offiziellen Steigerungsbogen (PDF/Papier). Die Boni in den Kräften und Fertigkeiten zu hinterlegen die man kauft statt in Grunddaten ist wohl Geschmacksache. 

Im offiziellen Bogen die Beschreibungen direkt bei den Jägerkräften dazuzuschreiben und die aktiven bzw. bereits gekauften anzukreuzen ist praktisch. Eventuell baue ich auch noch was ein (Steigerung, Ankreuzen der Jägerkräfte, Button der die Variablen für den offiziellen Bogen füllt und/oder umgekehrt, falls ich die 5 Charaktere nicht ggf. manuell übernehme). 

Allerdings habe ich mich immer noch nicht darum gekümmert wie das mit den Pull Requests funktioniert bzw. ob das auch im Web Frontend ohne Client geht, ich habe bisher nur Cut & Paste aus der RAW Darstellung in Roll20 oder VIM (Editor) hinein verwendet.
